test/unit/junitReporter.js

```diff
@@ -0,0 +1,91 @@
+var junitReportBuilder = require('junit-report-builder');
```

Comment:
Are you talking about the produced XML file? As I'm using the junit-report-builder module for the XML creation & file system write, it is not our task to test it: the junit-report-builder module should have such tests.